views/default/js/mentions/editors/ckeditor.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * CKEditor support for the mentions plugins
+ */
+define(function(require) {
+	var mentions = require('mentions/autocomplete');
+
+	require(['ckeditor'], function(CKEDITOR) {
```

Comment:
I'm not completely sure in fact. I made multiple quick iterations of this and I'm not sure if it's needed in this one.